### ISSUE TITLE
Training loss is decreasing while testing loss is increasing

### ISSUE THREAD

**Issue by ilovecv** [ZhiwenShao/PyTorch-JAANet#10]
Hi,

Thank you very much for sharing your codes. When I trained the model on BP4D dataset, I found the training loss is decreasing, both F1 score and Accuracy score are increasing. However, the testing loss is increasing, but the F1 score is increasing and the Accuracy score is decreasing. Is it normal? Did you have this problem? Thank you very much!

**Comment by ZhiwenShao** [comment]
I am sorry I have not encountered this issue. I guess there are maybe some errors about the preprocessing of your datasets. Maybe [this](https://github.com/ZhiwenShao/PyTorch-JAANet/issues/3) is a solution.

**Comment by ilovecv** [comment]
@ZhiwenShao Thank you very much for your reply. I am wondering if it is possible for you to share a training log with me? Thanks!

**Comment by ilovecv** [comment]
@ZhiwenShao May I know what's the meaning of the missing label used in your [AU_detection_evalv1](https://github.com/ZhiwenShao/PyTorch-JAANet/blob/154251ac287b9760fafb6826544a088116305c00/util.py#L27) function? You set missing_label = 9. If I understand correctly, the action units only contain 0 or 1 labels. Thanks!

**Comment by saiakhil19932** [comment]
@ilovecv did you use BP4D dataset which provided by fera 2015?

**Comment by ZhiwenShao** [comment]
Yes, we use the BP4D dataset in FERA 2015.

**Comment by rafikg** [comment]
> @ZhiwenShao May I know what's the meaning of the missing label used in your [AU_detection_evalv1](https://github.com/ZhiwenShao/PyTorch-JAANet/blob/154251ac287b9760fafb6826544a088116305c00/util.py#L27) function? You set missing_label = 9. If I understand correctly, the action units only contain 0 or 1 labels. Thanks!

I have the same question here, what is missing_label = 9. I am working on DISFA

**Comment by ZhiwenShao** [comment]
In the AU_detection_evalv1 function, missing_label = 9 is set for BP4D dataset. There is no missing label in the DISFA dataset. new_curr_actual = curr_actual[curr_actual != missing_label] has no influence on other datasets without missing labels such as DISFA dataset.
